Source/WebCore/style/StyleFontSizeFunctions.cpp

```diff
@@ -183,5 +184,19 @@ int legacyFontSizeForPixelSize(int pixelFontSize, bool shouldUseFixedDefaultSize
     return findNearestLegacyFontSize<float>(pixelFontSize, fontSizeFactors, mediumSize);
 }
 
+float adjustedFontSize(float size, float sizeAdjust, float xHeight)
+{
+    float aspectValue = xHeight / size;
+    return size * ( sizeAdjust / aspectValue );
+}
+
+float adjustedFontSize(float size, float sizeAdjust, const FontMetrics& metrics)
+{
+    if (!metrics.hasXHeight())
+        return size;
+
+    return adjustedFontSize(size, sizeAdjust, metrics.xHeight());
+}
+
 }
 }
```

Comment:
I know you didn't write this but this might be a good place to add closing-brace comments.

E.g.
```
} // namespace Foo
```